src/components/records.vue

```diff
@@ -0,0 +1,34 @@
+<script>
+import record from './record'
+import modal from './modal'
+import { clipboard } from 'electron'
+export default {
+  name: 'records',
+  data() {
+    return {
+      isModalVisible: this.showModal,
+      modalRecord: null
+    }
+  },
+  components: { record, modal },
+  props: ['records', 'showModal'],
+  methods: {
+    async copyToClipboard(record) {
+      clipboard.writeText(JSON.stringify(record, null, 2))
+    }
+  }
+}
+</script>
+
+<template lang="pug">
+  div.record-root
+    modal(v-if="isModalVisible" @close="isModalVisible = false" :record="modalRecord")
+    record.asd(v-for="record in records" :record="record" :tabindex="0" :rootname="'Document'" :indentLevel="1" v-on:dblclick.native ="isModalVisible = !isModalVisible; modalRecord = record; " @keydown.native.meta.8="$emit('deleteRecord', record)" @keydown.native.meta.67="copyToClipboard(record)")
```

Comment:
are you sure about `.asd`? :)

Reply:
how do we trigger delete and copy?

Reply:
I have no idea about asd :) Probably, was trying about something about styling and forgot to clean up my mess. Delete and copy should be working as you specified in issue details.